ambry-rest/src/main/java/com/github/ambry/rest/PublicAccessLogHandler.java

```diff
@@ -165,16 +178,24 @@ private void logHeaders(String tag, HttpMessage message, String[] headers) {
   private void logDurations() {
     long nowMs = System.currentTimeMillis();
     logMessage.append("bytes sent=").append(responseBytesSent).append(" ");
+    structuredLogMessage.put("bytesSent", String.valueOf(responseBytesSent));
     logMessage.append("duration=").append(nowMs - requestArrivalTimeInMs).append("ms ");
+    structuredLogMessage.put("duration", String.valueOf(nowMs - requestArrivalTimeInMs).concat("ms"));
     if (requestLastChunkArrivalTimeInMs != INIT_TIME) {
       logMessage.append("(chunked request receive=")
           .append(requestLastChunkArrivalTimeInMs - requestArrivalTimeInMs)
           .append("ms) ");
+      structuredLogMessage.put("chunkedRequestReceive",
```

Comment:
Here and at other places, do you think there's a value in keeping the error message same as before?

Like in this case keeping the message as "(chunked request receive="?

Reply:
Thanks for pointing this out! I changed the code such that when we actually print the log statements, we include the full contents of the old log message as a "message" field. This will certainly bloat the logs a bit, but for the time being should make it easier to read the logs for humans.